eng/pipelines/templates/stages/archetype-sdk-publish-js.yml

```diff
@@ -27,53 +27,47 @@ extends:
           dependsOn: ${{ parameters.BuildStageName }}
 
           jobs:
-            - deployment: Publish
-              environment: 'package-publish'
+            - job: PrePublish
+              variables:
+                - template: /eng/pipelines/templates/variables/image.yml
               pool:
-                name: azsdk-pool-mms-ubuntu-2004-general
-                image: azsdk-pool-mms-ubuntu-2004-1espt
+                name: $(LINUXPOOL)
+                image: $(LINUXVMIMAGE)
                 os: linux
+              steps:
+                - checkout: self
 
-              strategy:
-                runOnce:
-                  deploy:
-                    steps:
-                    - checkout: self
+                - download: current
+                  artifact: ${{ parameters.ArtifactName }}
+                  timeoutInMinutes: 5
 
-                    - download: current
-                      artifact: ${{ parameters.ArtifactName }}
-                      timeoutInMinutes: 5
+                - task: PowerShell@2
+                  inputs:
+                    filePath: '$(Build.SourcesDirectory)/eng/scripts/determine-js-release-tag.ps1'
+                    failOnStderr: true
+                    pwsh: true
+                    arguments: '-PackageJsonPath ${{ parameters.PackageJsonPath }}'
 
-                    - task: PowerShell@2
-                      inputs:
-                        filePath: '$(Build.SourcesDirectory)/eng/scripts/determine-js-release-tag.ps1'
-                        failOnStderr: true
-                        pwsh: true
-                        arguments: '-PackageJsonPath ${{ parameters.PackageJsonPath }}'
+                - pwsh: |
+                    Write-Host "Will deploy with tag of $(Tag)"
+                    Get-ChildItem "$(Pipeline.Workspace)/${{ parameters.ArtifactName }}" -Recurse -Force `
+                      | Where-Object { $_.Name -like "*.tgz" } `
+                      | Copy-Item -Destination "$(Build.ArtifactStagingDirectory)"
```

Comment:
What does moving do here if we aren't uploading the artifact?

Reply:
removed